packages/client/src/views/Organization/ListOfOrganizations.tsx

```diff
@@ -0,0 +1,187 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+import React from 'react'
+import { Frame } from '@opencrvs/components/lib/Frame'
+import { Header } from '@client/components/Header/Header'
+import { navigationMessages } from '@client/i18n/messages/views/navigation'
+import { constantsMessages } from '@client/i18n/messages'
+import { Navigation } from '@client/components/interface/Navigation'
+import { useIntl } from 'react-intl'
+import { Pagination } from '@opencrvs/components/lib/Pagination'
+import {
+  Content,
+  Link,
+  ListViewItemSimplified,
+  ListViewSimplified,
+  Stack,
+  Text
+} from '@opencrvs/components'
+import { useDispatch, useSelector } from 'react-redux'
+import { IStoreState } from '@client/store'
+import { ILocation } from '@client/offline/reducer'
+import { useHistory, useParams } from 'react-router'
+import { goToOrganizationList } from '@client/navigation'
+import { Activity } from '@opencrvs/components/lib/icons'
+import { Button } from '@opencrvs/components/lib/Button'
+
+const DEFAULT_PAGINATION_LIST_SIZE = 10
+export function ListOfOrganizations() {
+  const intl = useIntl()
+  const { index } = useParams()
+  const dispatch = useDispatch()
+  //
+  const getNewLevel =
+    (index: string) =>
+    (store: IStoreState): ILocation[] => {
+      let currentLevel = Object.values(
+        store.offline.offlineData.locations as { [key: string]: ILocation }
+      ).filter((s) => s.partOf === `Location/${index ?? '0'}`)
+      if (currentLevel.length === 0) {
+        currentLevel = Object.values(
+          store.offline.offlineData.offices as { [key: string]: ILocation }
+        ).filter((s) => s.partOf === `Location/${index ?? '0'}`)
+      }
+      return currentLevel
+    }
+
+  const levelOne = useSelector<IStoreState, any>(getNewLevel(index))
+  const totalNumber = levelOne.length
+  const [currentPageNumber, setCurrentPageNumber] = React.useState<number>(1)
+
+  const [links, setLink] = React.useState([
```

Comment:
I think you shouldn't store the breadcrumb in the component state, but instead, compute it from the current location id in the browser path every time the component renders using the locations data from the redux state.